paymentsheet/src/main/java/com/stripe/android/paymentsheet/paymentdatacollection/ach/USBankAccountFormViewModel.kt

```diff
@@ -477,55 +532,80 @@ internal class USBankAccountFormViewModel @Inject internal constructor(
     }
 
     private fun updatePaymentSelection(
-        linkAccountId: String,
+        resultIdentifier: ResultIdentifier,
         bankName: String?,
         last4: String?
     ) {
         if (bankName == null || last4 == null) return
 
         val paymentSelection = createNewPaymentSelection(
+            resultIdentifier = resultIdentifier,
             last4 = last4,
             bankName = bankName,
-            linkAccountId = linkAccountId,
         )
 
         _result.tryEmit(paymentSelection)
         shouldReset = true
     }
 
     private fun createNewPaymentSelection(
+        resultIdentifier: ResultIdentifier,
         last4: String,
         bankName: String,
-        linkAccountId: String,
     ): PaymentSelection.New.USBankAccount {
         val customerRequestedSave = customerRequestedSave(
             showCheckbox = args.showCheckbox,
             saveForFutureUse = saveForFutureUse.value
         )
```

Comment:
@carlosmuvi-stripe Do we keep the existing `Save for future use` behavior with Instant Debits?